_includes/pbc/all/install-features/202212.0/install-the-agent-assist-feature.md

```diff
@@ -0,0 +1,528 @@
+
+use Spryker\Yves\SessionRedis\Plugin\SessionAgentValidation\SessionRedisSessionAgentSaverPlugin;
+use Spryker\Yves\SessionRedis\Plugin\SessionAgentValidation\SessionRedisSessionAgentValidatorPlugin;
+use SprykerShop\Yves\SessionAgentValidation\SessionAgentValidationDependencyProvider as SprykerSessionAgentValidationDependencyProvider;
+use SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentSaverPluginInterface;
+use SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentValidatorPluginInterface;
+
+class SessionAgentValidationDependencyProvider extends SprykerSessionAgentValidationDependencyProvider
+{
+    /**
+     * @return \SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentSaverPluginInterface
+     */
+    protected function getSessionAgentSaverPlugin(): SessionAgentSaverPluginInterface
+    {
+        return new SessionRedisSessionAgentSaverPlugin();
+    }
+
+    /**
+     * @return \SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentValidatorPluginInterface
+     */
+    protected function getSessionAgentValidatorPlugin(): SessionAgentValidatorPluginInterface
+    {
+        return new SessionRedisSessionAgentValidatorPlugin();
+    }
+}
+```
+
+{% info_block warningBox "Warning" %}
+
+Apply the following changes only if session data is store in a file.
+
+{% endinfo_block %}
+
+**src/Pyz/Yves/SessionAgentValidation/SessionAgentValidationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\SessionAgentValidation;
+
+use Spryker\Yves\SessionFile\Plugin\SessionAgentValidation\SessionFileSessionAgentSaverPlugin;
+use Spryker\Yves\SessionFile\Plugin\SessionAgentValidation\SessionFileSessionAgentValidatorPlugin;
+use SprykerShop\Yves\SessionAgentValidation\SessionAgentValidationDependencyProvider as SprykerSessionAgentValidationDependencyProvider;
+use SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentSaverPluginInterface;
+use SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentValidatorPluginInterface;
+
+class SessionAgentValidationDependencyProvider extends SprykerSessionAgentValidationDependencyProvider
+{
+    /**
+     * @return \SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentSaverPluginInterface
+     */
+    protected function getSessionAgentSaverPlugin(): SessionAgentSaverPluginInterface
+    {
+        return new SessionFileSessionAgentSaverPlugin();
+    }
+
+    /**
+     * @return \SprykerShop\Yves\SessionAgentValidationExtension\Dependency\Plugin\SessionAgentValidatorPluginInterface
+     */
+    protected function getSessionAgentValidatorPlugin(): SessionAgentValidatorPluginInterface
+    {
+        return new SessionFileSessionAgentValidatorPlugin();
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Log in as an agent.
+2. Verify that the session key was added to a storage (Redis or File).
```

Comment:
It might be a good idea to say how exactly to do this for file or Redis. We probably need two separate verification sections then.